create_framework/http-foundation.rst

```diff
@@ -121,6 +121,17 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    {
+        "require": {
+            "symfony/http-foundation": "^2.7"
+    }
+
+The code-block shows the content of the ```composer.json```, but the version may be different.
```

Comment:
This should only be two backticks (two before and two after).

And I would reword it a bit:

```rst
The code block shows the content of the ``composer.json`` file (the actual
version may vary).
```